### ISSUE TITLE
Typography tokens not correctly applied to shared styles

### ISSUE THREAD

**Issue by mzilverberg** [six7/figma-tokens#993]
**Description**
If I create typography styles from my tokens, every style is created as 'Inter 12px' instead of applying values from my tokens. Additionally, if I update my tokens, the styles are not updated accordingly.

**To Reproduce**
1. Press 'create styles' from the plugin

**Expected behavior**
I expect the token value (or its references) to be applied to the shared styles. This definitely _has_ worked before. I recently noticed it didn't work as expected anymore, but couldn't find similar reported issues.

**Screenshots**
<img width="404" alt="typography token" src="https://user-images.githubusercontent.com/4511290/177995255-64166cea-f22a-481f-8730-dcdd8e8c91b9.png">

<img width="482" alt="wrong typography" src="https://user-images.githubusercontent.com/4511290/177994874-4cc38fa6-e709-4659-ac8e-ecacbc8e4ef3.png">

**JSON (snippet)**
```json
{
    "Core": {
      "FontFamily": {
        "value": "PostNL",
        "type": "fontFamilies"
      },
      "FontSize": {
        "XXL": {
          "value": "28px",
          "type": "fontSizes"
        },
        "3XL": {
          "value": "36px",
          "type": "fontSizes"
        },
        "4XL": {
          "value": "48px",
          "type": "fontSizes"
        }
      },
      "LineHeight": {
        "XS": {
          "value": "110%",
          "type": "lineHeights",
          "description": "1.1"
        },
        "M": {
          "value": "130%",
          "type": "lineHeights",
          "description": "1.3"
        }
      },
      "FontWeight": {
        "Light": {
          "value": "300",
          "type": "fontWeights",
          "description": "Light"
        },
        "Regular": {
          "value": "400",
          "type": "fontWeights",
          "description": "Regular"
        }
      }
    },
    "Semantic": {
      "Text": {
        "Heading": {
          "XXL": {
            "value": {
              "fontFamily": "{FontFamily}",
              "fontWeight": "{FontWeight.Light}",
              "lineHeight": "{LineHeight.XS}",
              "fontSize": "{FontSize.4XL}",
              "letterSpacing": "0%",
              "paragraphSpacing": "0",
              "textDecoration": "none",
              "textCase": "none"
            },
            "description": "",
            "type": "typography"
          },
          "XL": {
            "value": {
              "fontFamily": "{FontFamily}",
              "fontWeight": "{FontWeight.Light}",
              "lineHeight": "{LineHeight.XS}",
              "fontSize": "{FontSize.3XL}",
              "letterSpacing": "0%",
              "paragraphSpacing": "0",
              "textDecoration": "none",
              "textCase": "none"
            },
            "description": "",
            "type": "typography"
          },
          "L": {
            "value": {
              "fontFamily": "{FontFamily}",
              "fontWeight": "{FontWeight.Regular}",
              "lineHeight": "{LineHeight.M}",
              "fontSize": "{FontSize.XXL}",
              "letterSpacing": "0%",
              "paragraphSpacing": "0",
              "textDecoration": "none",
              "textCase": "none"
            },
            "description": "",
            "type": "typography"
          }
        }
      }
    },
    "$themes": []
  }
```

**Comment by mzilverberg** [comment]
Additional information, I was able to reproduce this bug:

- in a new file
- with a different font (Roboto)

**Comment by six7** [comment]
Make sure you set your fontWeight to the same value that you can choose in Figma. That is often not `400` but rather `Regular` or `Light`.

Here's an issue on why this is happening: https://github.com/six7/figma-tokens/issues/807

TLDR: Figma requires us to set font family and weight with the same string to be used as if you'd set the property in Figma itself.

**Comment by mzilverberg** [comment]
Ah makes sense! And I mean both the explanation and the fact that this stopped working for me a while ago, as I switched from the values `Regular` etc to numeric values. Will revert this. Thanks for your quick reply!

**Comment by dabowman** [comment]
Noting for anyone else that comes here that the fontWeight is case sensitive! if you're passing Figma `regular` it will fail where `Regular` will pass. They all need to be in title case.
